### ISSUE TITLE
Ability to chose namespace for a multi cluster app target

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#17913]
Today multi cluster app target allows you to chose a project as a destination for your app deployment, and the namespace gets auto-created. We should let user supply a namespace name (optional) as  part of target. 
Not targeting any release for now, just having this enhancement request open to keep track.

**Comment by polivbr** [comment]
Yes, please. I would love to use multi-cluster apps, but this is a showstopper for me.

**Comment by tzilist** [comment]
This would be great! We already have clusters set up in a specific way (which works for us). Not being able to specify the namespace will cause quite a bit of headache when migrating over to Rancher 2.x

**Comment by yasker** [comment]
It impacts the Longhorn deployment in multiple clusters. See https://github.com/rancher/rancher/issues/19803 for details.

**Comment by Hitesh-Agrawal** [comment]
Any update on this feature , this is definitely a show stopper for us.

**Comment by ccckblaze** [comment]
yes, a more common issue is some helm chart developer likes to use a fixed namespace instead of custom namespace, the compability is limited because of this.

**Comment by mitchellmaler** [comment]
Based on the [documentation ](https://rancher.com/docs/rancher/v2.x/en/deploy-across-clusters/multi-cluster-apps/) we should be looking at Fleet as the way to deploy charts to multiple clusters.
